<h1 align="center"> üå± Alena Koshkareva üå± </h1>

[![Telegram](https://img.shields.io/badge/Telegram--brightgreen?style=social&logo=Telegram)](https://t.me/a_koshkareva)
[![Email](https://img.shields.io/badge/Email-alyona.koshkareva@gmail.com-white?style=flat&logo=gmail&logoColor=red&labelColor=white)](mailto:alyona.koshkareva@gmail.com)
[![View CV](https://img.shields.io/badge/View%20CV-white?style=flat&logo=google-drive&logoColor=black&labelColor=white)](https://drive.google.com/file/d/1jEQiMQRBDlIns_KI_AEGSTlOgZk9uYkO/view?usp=sharing)

I'm a junior bioinformatician, with hands-on experience in working with biological data, scripting, and molecular biology.

### üî¨ Background  
I hold a BSc in Biology (Biophysics) and have completed core coursework in the professional program Bioinformatics for Biologists at the Bioinformatics Institute (2024‚Äì2025).
(Note: I have completed key modules but have not received a full program certificate)

üìå Looking for opportunities as a junior bioinformatician or intern to grow through real-world challenges and teamwork.


### üõ†Ô∏è Skills

**Bioinformatics tools & workflows:**  
Worked with biological data formats (FASTA, FASTQ, GFF/GTF), performed sequence alignment using **BLAST**, multiple sequence alignment with **Clustal Omega** and **MAFFT**, and gene annotation via **Ensembl** and **NCBI** databases.  
Performed variant interpretation using **ClinVar** and post-processing with **BEDTools**.  
Used trimming tools like **ClipKit** and **TrimAl** to refine alignments and constructed phylogenetic trees using **FastTree** and **IQ-TREE**.  
Visualized tree topologies in **R** using the **treedataverse** suite (e.g., `ggtree`).  
Also experienced in protein structure prediction with **AlphaFold** and scripting data pipelines with **Biopython**, visualizing results using **pandas**, **matplotlib**, and **seaborn**.

**Programming & scripting:**  
Python (pandas, matplotlib, Biopython), Jupyter Notebooks, Bash scripting, R (tidyverse), RStudio, Git,  LaTeX.

<p align="left">
  <img src="https://img.shields.io/badge/Python-3776AB?style=for-the-badge&logo=python&logoColor=white"/>
  <img src="https://img.shields.io/badge/Biopython-3776AB?style=for-the-badge&logo=python&logoColor=white"/>
  <img src="https://img.shields.io/badge/NumPy-013243?style=for-the-badge&logo=numpy&logoColor=white"/>
  <img src="https://img.shields.io/badge/Pandas-150458?style=for-the-badge&logo=pandas&logoColor=white"/>
  <img src="https://img.shields.io/badge/Matplotlib-007ACC?style=for-the-badge&logo=plotly&logoColor=white"/>
  <img src="https://img.shields.io/badge/Seaborn-76B900?style=for-the-badge&logo=python&logoColor=white"/>
  <img src="https://img.shields.io/badge/Jupyter-F37626?style=for-the-badge&logo=Jupyter&logoColor=white"/>
  <img src="https://img.shields.io/badge/R-276DC3?style=for-the-badge&logo=r&logoColor=white"/>
  <img src="https://img.shields.io/badge/RStudio-75AADB?style=for-the-badge&logo=RStudio&logoColor=white"/>
  <img src="https://img.shields.io/badge/Bash-4EAA25?style=for-the-badge&logo=gnu-bash&logoColor=white"/>
</p>
<p align="left">
  <img src="https://img.shields.io/badge/Git-F05032?style=for-the-badge&logo=git&logoColor=white"/>
  <img src="https://img.shields.io/badge/LaTeX-008080?style=for-the-badge&logo=latex&logoColor=white"/>
  <img src="https://img.shields.io/badge/Overleaf-47A141?style=for-the-badge&logo=Overleaf&logoColor=white"/>
  <img src="https://img.shields.io/badge/Notion-000000?style=for-the-badge&logo=notion&logoColor=white"/>
</p>
